src/rf/apps/home/views.py

```diff
@@ -205,3 +205,25 @@ def _get_layers(request, crit=None):
     """
     models = _get_layer_models(request, crit)
     return [m.to_json() for m in models]
+
+
+@api_view
+@accepts('POST')
+@csrf_exempt
+def update_status(request):
```

Comment:
I think this endpoint should require logged in users, and should be forbidden to everyone except superusers. Dummy API token authentication is implemented to be able to test this. Appending `?api_key=<user_id>` to the end of any API URL will set `request.user` to the correct user.